public/visualization.component.ts

```diff
@@ -159,46 +236,47 @@ export class VisualizationComponent {
             if (colorBy === "gender") {
                 switch (entry.gender) {
                     case "Male":
-                        myColor = "cornflowerblue";
+                        myColor = MALE_COLOR;
```

Comment:
I think it might be better to rename these to `COLOR_MALE`, etc. That way autocomplete would be useful when you type in "COLOR_"